src/metrics.ts

```diff
@@ -443,23 +443,23 @@ export function getMetrics(
       labelNames: ['topic']
     }),
     /** Tracks distribution of recv msgs by duplicate, invalid, valid */
-    msgReceivedInvalidTotal: register.gauge<{ topic: TopicLabel }>({
-      name: 'gossipsub_msg_received_invalid_total',
+    prevalidationInvalidTotal: register.gauge<{ topic: TopicLabel }>({
+      name: 'gossipsub_pre_validation_invalid_total',
       help: 'Total count of invalid messages received',
       labelNames: ['topic']
     }),
-    msgReceivedValidTotal: register.gauge<{ topic: TopicLabel }>({
-      name: 'gossipsub_msg_received_valid_total',
+    prevalidationValidTotal: register.gauge<{ topic: TopicLabel }>({
+      name: 'gossipsub_pre_validation_valid_total',
       help: 'Total count of valid messages received',
       labelNames: ['topic']
     }),
-    msgReceivedDuplicateTotal: register.gauge<{ topic: TopicLabel }>({
-      name: 'gossipsub_msg_received_duplicate_total',
+    prevalidationDuplicateTotal: register.gauge<{ topic: TopicLabel }>({
+      name: 'gossipsub_pre_validation_duplicate_total',
       help: 'Total count of duplicate messages received',
       labelNames: ['topic']
     }),
-    msgReceivedUnknownStatusTotal: register.gauge<{ topic: TopicLabel }>({
-      name: 'gossipsub_msg_received_unknown_status_total',
+    prevalidationUnknownTotal: register.gauge<{ topic: TopicLabel }>({
+      name: 'gossipsub_pre_validation_unknown_status_total',
```

Comment:
hmm is the duplicate metric on lodestar or here? we should try to not break metrics on upstream libraries.

Reply:
this metric was introduced in the previous PR https://github.com/ChainSafe/js-libp2p-gossipsub/pull/460/files#diff-a210f4dc1ff876b56a2a9db45167223854fcbf9b9dc6bd0b4e92433582a3d6f8R462

no one uses that metric for now @dapplion